tools/bash.py

```
@@ -1,257 +1,230 @@
         raise ToolError("no command provided.")
 
     async def _run_command(self, command: str):
         """Execute a command in the shell."""
+        output = ""
         try:
-            prompt = read_prompt_from_file(PROMPT_FILE, command)
-            # rr(f"[red]{'*' * 80}[/red]")
+            if self.display:
+                self.display.add_message("user", f"Processing command: {command}")
+
+            prompt = read_prompt_from_file(BASH_PROMPT_FILE, command)
```

Ruff (0.8.2)

Undefined name `BASH_PROMPT_FILE`

(F821)